test/blip/components/CalendarContainer.test.js

```diff
@@ -48,19 +49,19 @@ describe('CalendarContainer', () => {
   var props = {
     bgClasses: {
       'very-low': {
-        boundary: 60,
+        boundary: DEFAULT_BG_BOUNDS[MGDL_UNITS].veryLow,
       },
       'low': {
-        boundary: 80,
+        boundary: DEFAULT_BG_BOUNDS[MGDL_UNITS].targetLower,
       },
       'target': {
-        boundary: 180,
+        boundary: DEFAULT_BG_BOUNDS[MGDL_UNITS].targetUpper,
       },
       'high': {
-        boundary: 200,
+        boundary: DEFAULT_BG_BOUNDS[MGDL_UNITS].veryHigh,
       },
       'very-high': {
-        boundary: 300,
+        boundary: 600,
```

Comment:
No biggie, but we should probably use `BG_CLAMP_THRESHOLD[MGDL_UNITS]` here and in the other tests for consistency.